File: bubble_sort.js
const bubble_sort=(arr)=>{
    for(let i=0;i<arr.length;i++){
        for(let j=0;j<arr.length-i;j++){
            if(arr[j]>arr[j+1]){
                let temp=arr[j];
                arr[j]=arr[j+1];
                arr[j+1]=temp;
            }
        }
    }
    return arr;
}

const arr=[1,2,3,5,6,8,10,9,7,4]
const res=bubble_sort(arr)
console.log(res)

File: bubble_sort.py
def bubble_sort(arr):
    n=len(arr)
    for j in range(0,n-1):
        for i in range(0,n-j-1):
            if arr[i]>arr[i+1]:
                arr[i],arr[i+1]=arr[i+1],arr[i]
    
    return arr

arr=[64, 34, 25, 12, 22, 11, 90]

print(bubble_sort(arr))